### ISSUE TITLE
Previous and next buttons do not indicate whether the screen reader user is at the beginning or end of the carousel

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7191]
### Where To Find Bug
Home page

### Expected Behaviour
The screen reader should announce when the carousel gets to the first or last element

### Actual Behaviour
The screen reader gives no indication of when the carousel gets the beginning nor the end

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Tab through the page until you get to the carousel next or previous button
- Notice that the screen reader only says "previous button" or "next" button once no matter how many times you press that button

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
